themes/Casio-color-theme.json

```diff
@@ -691,6 +691,130 @@
 			"settings": {
 				"foreground": "#F49E8B",
 			}
-		}, 
+		},
+		// Typescript
+		{
+			"scope": [
+				"source.ts keyword.control",
+				"keyword.operator.ternary.ts",
+				"keyword.operator.new.ts",
+				"keyword.operator.logical.ts",
+				"keyword.operator.relational.ts",
+				"keyword.operator.comparison.ts",
+				"storage.modifier.ts",
+			],
+			"settings": {
+				"foreground": "#e67574"
+			},
+		},
+		{
+			"scope": [
+				"punctuation.terminator.statement.ts",
+				"source.ts keyword.operator",
+				"source.ts punctuation.accessor",
+				"support.type.object.module.ts",
+				"constant.language.import-export-all.ts"
+			],
+			"settings": {
+				"foreground": "#98999b",
+			}
+		},
+		{
+			"name": "Strings",
+			"scope": [
+				"string.quoted.ts",
+				"string.quoted.single.ts",
+				"string.template.ts"
+			],
+			"settings": {
+				"foreground": "#4cbe9a"
+			}
+		},
+		{
+			"name": "Comments",
+			"scope": [
+				"source.ts comment.line.double-slash.ts",
+				"comment.block.ts"
+			],
+			"settings": {
+				"foreground": "#6c7277"
+			}
+		},
+		{
+			"name": "Comment documentation",
+			"scope": "comment.block.documentation.ts",
+			"settings": {
+				"foreground": "#8e9499"
+			}
+		},
+		{
+			"name": "Variables",
+			"scope": "variable.other.constant.ts",
+			"settings": {
+				"foreground": "#82cab3"
+			}
+		},
+		{
+			"scope": "support.type.primitive.ts",
+			"settings": {
+				"foreground": "#D4D4D4",
+				"fontStyle": "underline"
+			}
+		},
+		{
+			"scope": "entity.name.type.ts",
```

Comment:
This value has a conflict with the color of the interfaces at: https://github.com/ChirujanoCodding/Casio-theme/blob/main/themes/Casio-color-theme.json#L101